### PR TITLE
icu: Bumped to v57.1

### PR DESCRIPTION
**Pull request by Xotic750**
Signed-off-by: Graham Fairweather xotic750@gmail.com

### CODE REVIEW

File: libs/icu/Makefile [hunk 1]
```diff
@@ -41,23 +41,36 @@ define Package/icu
...
+	--enable-release \
+	--enable-shared \
+	--enable-static \
+	--enable-draft \
+	--enable-renaming \
+	--disable-tracing \
+	--disable-extras \
+	--enable-dyload \
+	--enable-layout \
+	--enable-layoutex \
```
Comment (Noltari) [inline libs/icu/Makefile]: Please, remove this backslash

File: libs/icu/Makefile [hunk 2]
```diff
@@ -41,23 +41,36 @@ define Package/icu
...
+				
+CONFIGURE_ARGS += $(CONFIGURE_ARGS_ICU) \
+	--disable-tools \
+	--disable-tests \
+	--disable-samples \
+	--with-cross-build="$(HOST_BUILD_DIR)"
+	
+HOST_CONFIGURE_ARGS += $(CONFIGURE_ARGS_ICU)
+
 define Build/Configure
```
Comment (Noltari) [inline libs/icu/Makefile]: Can't we remove this by using "CONFIGURE_CMD = runConfigureICU Linux/gcc"?

File: libs/icu/Makefile [hunk 3]
```diff
@@ -41,23 +41,36 @@ define Package/icu
...
-		--disable-tools \
-		--disable-tests \
-		--disable-samples \
-		--with-cross-build="$(HOST_BUILD_DIR)" \
+	(cd $(PKG_BUILD_DIR) ; \
+		./runConfigureICU Linux/gcc $(CONFIGURE_ARGS) \
+	)
+endef
+
+define Host/Configure
```
Comment (Noltari) [inline libs/icu/Makefile]: Can't we remove this by using "HOST_CONFIGURE_CMD = runConfigureICU"?

File: libs/icu/Makefile (hunk 1 above)
Comment (Xotic750) [inline libs/icu/Makefile]: Is there some kind of syntax checker to spot things like this as there is no error or warning when building?

File: libs/icu/Makefile (hunk 2 above)
Comment (Xotic750) [inline libs/icu/Makefile]: Perhaps, I was not aware of it.

Comment (Xotic750) [inline libs/icu/Makefile]: I am looking at using this but I have been unable to get it to compile as the args do not seem to get passed. Not sure but `configure` looks hard coded in _host-build.mk_?
```
define Host/Configure/Default
	$(if $(HOST_CONFIGURE_PARALLEL),+)(cd $(HOST_BUILD_DIR)/$(3); \
		if [ -x configure ]; then \
			$(CP) $(SCRIPT_DIR)/config.{guess,sub} $(HOST_BUILD_DIR)/$(3)/ && \
			$(2) \
			$(HOST_CONFIGURE_CMD) \
			$(HOST_CONFIGURE_VARS) \
			$(HOST_CONFIGURE_ARGS) \
			$(1); \
		fi \
	)
endef
```

Comment (Xotic750) [inline libs/icu/Makefile]: I have made several attempts at using this but I haven't managed to get it to work yet.

Comment (Xotic750) [inline libs/icu/Makefile]: At this point I am going to have to admit defeat. What we have here is working even if not optimal/ideal code. If someone can give me some pointers to getting this suggestion working then I will look at it again.

**Comment by champtar** [comment]
@Noltari can you provide code exemple to @Xotic750 ?
@mirko please review

**Comment by diizzyy** [comment]
@Xotic750 
Your patch have been superseded, do you have anything to add to the new pull request?
Please close this one.

**Comment by mirko** [comment]
If anyone feels like taking care of the ICU package, feel free to take over.

**Comment by Xotic750** [comment]
I have nothing new to add. Thanks.

**Pull request closed without merging**